Makefile

```diff
@@ -1,10 +1,16 @@
-.PHONY: test build clean
+.PHONY: test build build-arwen clean
 
 clean:
 	go clean -cache -testcache
 
 build:
 	go build ./...
 
-test:
+build-arwen:
+	go build -o ./cmd/arwen/arwen ./cmd/arwen
+	cp ./cmd/arwen/arwen ./ipc/tests
+
+test: build-arwen run-tests
+
+run-tests:
```

Comment:
Add dependency on `clean` as well (i.e. `run-tests: clean`). I forgot to add it to `test`.

Reply:
Added.